examples/plot_optic_radiations.py

```diff
@@ -55,39 +83,100 @@
+#
 #     bundles = bundles + abd.BundleDict()
+#
+# In this case, we will skip this and generate just the OR.
+
 
-brain_mask_definition = LabelledImageFile(
-    suffix="seg",
-    filters={"scope": "freesurfer"},
-    exclusive_labels=[0])
+#############################################################################
+# Define GroupAFQ object
+# ----------------------
+# HBN POD2 have been processed with qsiprep [5]_. This means that a brain mask
+# has already been computer for them. As you can see in other examples, these
+# data also have a mapping calculated for them, which can also be incorporated
+# into processing. However, in this case, we will let pyAFQ calculate its own
+# SyN-based mapping so that the `combine_bundle` method can be used below to
+# create a montage visualization.
+#
+# For tractography, we use CSD-based probabilistic tractography seeding
+# extensively (`n_seeds=4` means 81 seeds per voxel) but only within the ROIs
+# and not throughout the white matter. This is controlled by passing
+# `"seed_mask": RoiImage()` in the `tracking_params` dict. The custom bundles
+# are passed as `bundle_info=bundles`. The call to `my_afq.export_all()`
+# initiates the pipeline.
+
+brain_mask_definition = ImageFile(
+    suffix="mask",
+    filters={'desc': 'brain',
+             'space': 'T1w',
+             'scope': 'qsiprep'})
 
 my_afq = GroupAFQ(
-    bids_path=op.join(
-        afd.afq_home,
-        'stanford_hardi'),
+    bids_path=study_dir,
+    preproc_pipeline="qsiprep",
+    output_dir=op.join(study_dir, "derivatives", "afq_or"),
     brain_mask_definition=brain_mask_definition,
-    tracking_params={"n_seeds": 3,
+    tracking_params={"n_seeds": 4,
                      "directions": "prob",
                      "odf_model": "CSD",
                      "seed_mask": RoiImage()},
     bundle_info=bundles)
 
 my_afq.export_all()
 
-if len(aus.SegmentedSFT.fromfile(my_afq.export("clean_bundles")[
-        "01"]).get_bundle("L_OR").streamlines) > 1:
-    #  create bundle montage and bundle combination
-    #  across subject/session in MNI
+#############################################################################
+# Visualize a montage
+# ----------------------
+# One way to examine the output of the pyAFQ pipeline is by creating a montage
+# of images of a particular bundle across a group of participants (or, in this
+# case, the one participant that was analyzed).
+#
+# To see whether this is doable, we first use our SegmentedSFT class to query
+# the "L_OR" bundle. After verifying that streamlines were found, we ask for a
+# montage with an axial view and display that. After creating the montage, we
+# can also ask for an interactive browser-based view of the bundles to be
+# displayed.
+
+if len(aus.SegmentedSFT.fromfile(
+    my_afq.export("clean_bundles")["NDARAA948VFH"]).get_bundle(
+        "L_OR").streamlines) > 1:
     montages = my_afq.montage("L_OR", (1, 1), "Axial")
     my_afq.combine_bundle("L_OR")
```

Comment:
@36000 :  why is `combine_bundle` called here? At this point, the image has presumably already been created.

Reply:
This is a smoke test for that function. It's the only place it is called. It could probably be moved into it's own, more comprehensive test using example streamlines but it would not be trivial, as it is attached and enmeshed with the GroupAFQ object.

Reply:
I'd rather not include this test here. I think that the examples should primarily serve as guidance for users getting acquainted with the software. Admittedly, we need to do some work in some of the other examples, to move them in that direction.